docs/vectorizer/worker.md

```diff
@@ -87,94 +170,114 @@ On your local machine:
    - Docker: `docker compose exec -it db psql`
    - psql:  `psql postgres://postgres:postgres@localhost:5432/postgres`
 
-### Standalone vectorizer worker with Docker
+For more configuration options, see [Advanced configuration options](#advanced-configuration-options) below.
 
-The `timescale/pgai-vectorizer-worker` docker image supplies the pgai vectorizer worker.
-You use this image to run vectorizers on any self-hosted Postgres database that has the
-pgai extension activated.
+# Configure the vectorizer worker
 
-On your local machine:
+Most users of the vectorizer worker will need to set the following configuration options:
 
-1. **Run the vectorizer worker**
+- [Setting the database connection string](#setting-the-database-connection-string)
+- [Setting API keys for embedding providers through environment variables (or .env file)](#setting-api-keys-through-environment-variables-or-env-file)
 
-   For self-hosted, you run a pgai vectorizer worker to automatically create embedding from the data in your
-   database using [vectorizers you defined previously](/docs/vectorizer.md#define-a-vectorizer).
+Other, advanced configuration options are available, see [Advanced configuration options](#advanced-configuration-options) below.
 
-   Start the vectorizer worker:
-   ```
-   docker run timescale/pgai-vectorizer-worker:{tag version} --db-url <DB URL>
-   ```
+## Setting the database connection string
 
-### Install vectorizer worker as a python package
+The vectorizer worker needs to know how to connect to your database. You can do this by setting the `-d` command line argument or the `PGAI_VECTORIZER_WORKER_DB_URL` environment variable.
 
-On your local machine:
+For example, if you are using a local Postgres database, you can set the database connection string as follows:
 
-1. **Install [pgai](https://pypi.org/project/pgai/) from PyPI**
+```
+pgai vectorizer worker -d postgres://postgres:postgres@localhost:5432/postgres
+```
 
-   ```shell
-   pip install pgai
-   ```
 
-   The vectorizer worker, `pgai vectorizer worker` is now in your `$PATH`.
+## Setting API keys through environment variables (or .env file)
 
-1. **Run the vectorizer worker**
+If you are using an embedding provider that requires an API key (which most do),
+you can set the API key through an environment variable or a .env file. We recommend
+using a .env file to set the API key.
```

Comment:
```suggestion
you can set the API key through an environment variable or a .env file.
```

Do we recommend that? I feel like for a real deployment I'd always use env vars while locally a .env file makes sense.